chapters/3-measurement.tex

```diff
@@ -229,8 +324,8 @@ \subsection{Defining study comparisons using master data}
 to exploit events that occurred in the past.
 Therefore, these methods often use either secondary data,
 including administrative data or other classes of routinely-collected information,
-and it is important that your data map documents 
-how that data is merged to any other data.
+and it is important that your data linkage table documents 
+how this data can be linked to the rest of the data in your project.
 
 
```

Comment:
I suggest to cut a LOT of what follows here, up through time periods. I can review more closely but the specific designs is now, I think, too much for this chapter. @mariaruth @luizaandrade

Reply:
Done in eea94d40c64696c1997d91673a9d49c41c1584a2 @kbjarkefur